lightning/src/util/config.rs

```diff
@@ -228,6 +243,7 @@ impl_writeable_tlv_based!(ChannelConfig, {
 	(4, announced_channel, required),
 	(6, commit_upfront_shutdown_pubkey, required),
 	(8, forwarding_fee_base_msat, required),
+	(10, max_balance_dust_htlc_msat, (default_value, 5_000_000)),
```

Comment:
This should also be odd - old versions should have no problem reading a config without this set, and will function just fine without it, they simply wont enforce the new rules.